### PR TITLE
Use HTTPS for more domains

### PR DESCRIPTION
**Pull request by mathiasbynens**
* benchmarkjs.com
* jsperf.com
* mths.be
* mathiasbynens.be

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 
 ## Documentation
 
-* [API Documentation](http://benchmarkjs.com/docs)
+* [API Documentation](https://benchmarkjs.com/docs)
 
 ## Download
 
- * [Development source](https://raw.github.com/bestiejs/benchmark.js/v2.1.0/benchmark.js)
+ * [Development source](https://raw.githubusercontent.com/bestiejs/benchmark.js/2.1.0/benchmark.js)
```
Comment (mathiasbynens) [inline README.md]: This URL 404’d. The tag didn’t exist yet, so I’ve created it. Also we seem to use the `2.1.0` format (no `v` prefix).

**Comment by mathiasbynens** [comment]
214f986b9de6d858532dfd7fd97c32cc47e58593 :open_mouth:

**Comment by jdalton** [comment]
> 214f986b9de6d858532dfd7fd97c32cc47e58593 :open_mouth: 

Odd that it stripped your author credit from it. You can see others where the author bit is preserved. The removal was not intentional. It can be restored I believe with some git-fu.

**Comment by mathiasbynens** [comment]
That’s alright, I don’t care. All the history rewriting + force-pushing is just a little weird.

**Comment by jdalton** [comment]
Rebase is the best base :)
It let those changes slip in before the 2.1.0 release.

**Pull request merged**